openxr/examples/vulkan.rs

```diff
@@ -409,27 +443,41 @@ fn main() {
                 continue;
             }
 
+            // Block until the previous frame is finished displaying, and is ready for another one.
+            // Also returns a prediction of when the next frame will be displayed, for use with
+            // predicting locations of controllers, viewpoints, etc.
             let xr_frame_state = frame_wait.wait().unwrap();
+            // Must be called before any rendering is done!
             frame_stream.begin().unwrap();
 
             if !xr_frame_state.should_render {
                 frame_stream
                     .end(
                         xr_frame_state.predicted_display_time,
-                        xr::EnvironmentBlendMode::OPAQUE,
+                        environment_blend_mode,
                         &[],
                     )
                     .unwrap();
                 continue;
             }
 
             if swapchain.is_none() {
+                // Now we need to find all the viewpoints we need to take care of! For a stereo
+                // headset, this should be 2. Similarly, for an AR phone, we'll need 1, and a VR
+                // cave could have 6, or even 12!
```

Comment:
I feel like this is a little misleading, as it could be understood to mean that the number of views is unpredictable, whereas in fact it's a property of the view configuration type--so here we're absolutely guaranteed that there are exactly two.